recipe/meta.yaml

```diff
@@ -63,26 +62,20 @@ outputs:
         - coveralls >=1.8.0
         - doxygen >=1.9.8
         - eups >=2.2.8
-        - flake8 ==7.1.0
-        - libblas * *openblas
-        - openblas
+        - flake8 ==7.1.1
         - make >=4.3
         - meson >=1.3.0
-        - nose >=1.3.7
         - pep8-naming >=0.13.3
         - pip >=23.3.2
-        - pkg-config >=0.29.2
```

Comment:
Why was pkg-config removed?

Reply:
I think it's in the rubin-env section.

Reply:
I saw it only for the `-rsp` output?

Reply:
You are right. @roceb it should be here and should be removed from the rubin-env-rsp section (the comment resolved below means this is half done because I hadn't spotted the removal up here, and only saw the addition in the wrong place down in rubin-env-rsp.